site/_data/i18n/nav/side_nav.yml

```
@@ -10,6 +10,9 @@ blog:
   en: 'Blog'
   es: 'Blog'
 
+100:
+  en: 'Chrome 100'
+
 releases:
   en: 'Releases'
-  es: 'Versiones'
\ No newline at end of file
+  es: 'Versiones'
```

Drive-by:

```suggestion
  es: 'Lanzamientos'
```

See, for example, https://es.wikipedia.org/wiki/Ciclo_de_vida_del_lanzamiento_de_software.

---

We are not supposed to make any changes in the existing content hence leaving it as it is!